design/user_types.go

```diff
@@ -129,3 +129,10 @@ var UpdateTrackerQueryAlternatePayload = a.Type("UpdateTrackerQueryAlternatePayl
 	})
 	a.Required("query", "schedule", "trackerID")
 })
+
+// IdentityData represents an identified user object
+var IdentityData = a.Type("IdentityData", func() {
```

Comment:
You cannot have the attributes like `fullName` and `imageURL` as members of the `data` object. Instead create an `attributes`  object like so:

```json
{
  "data": {
    "id": "WHATEVER",
    "type": "identities",
    "attributes": {
      "fullName": "John Doe",
      "imageURL": "http://....."
    }
  }
}
```

**NOTICE: The `type` member is new and has the plural form of the indentity.**